server/schedulers/balance_test.go

```diff
@@ -314,22 +314,22 @@ func (s *testBalanceLeaderSchedulerSuite) TestBalanceFilter(c *C) {
 	s.tc.AddLeaderStore(4, 16)
 	s.tc.AddLeaderRegion(1, 4, 1, 2, 3)
 
-	testutil.CheckTransferLeader(c, s.schedule()[0], operator.OpBalance, 4, 1)
+	testutil.CheckTransferLeader(c, s.schedule()[0], operator.OpKind(0), 4, 1)
```

Comment:
opRegion?

Reply:
It doesn't matter since the original `OpBalance` doesn't work.